solutionbox/code_free_ml/mltoolbox/code_free_ml/data/local_predict.py

```diff
@@ -0,0 +1,251 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
I used that name at first, but then I thought it might be confusing, so I named it local_predict